docs/changes/cosmology/15484.api.rst

```diff
@@ -0,0 +1,3 @@
+``Cosmology`` and its subclasses are now frozen ``dataclass`` objects.
+See the Python documentation for more information on ``dataclass`` objects: how they
```

Comment:
I think linking https://docs.python.org/3/library/dataclasses.html should be safe here ?

Reply:
@pllim, thoughts? I know we try to minimize links in our changelog

Reply:
I'd say that because there is a What's New entry then the change log entry can be shorter.

Reply:
Done!